sysconfigs/repos.yaml

```
@@ -1 +1,2 @@
 repos:
+  test addition
```

```suggestion
  test addition advanced
```
This is a bug, you need to change this